packages/wallet-adapter-core/src/new/registry.ts

```diff
@@ -0,0 +1,62 @@
+  deeplinkProvider: "nightly://v1?network=aptos&url=",
+};
+
+export interface DefaultRegistryConfig {
+  /**
+   * Aptos Connect is included by default.
+   * You can pass a custom configuration, or `false` to disable it.
+   */
+  aptosConnect?: AptosConnectWalletConfig | false;
+}
+
+export type RegistryWallet = DiscoverableWallet | AptosStandardWallet;
+
+/**
+ * Default registry of AIP-62 wallet standard wallets.
+ * This list is used to show supported wallets even if they are not installed on the user machine.
+ */
+export function makeDefaultRegistry({ aptosConnect = {} }: DefaultRegistryConfig = {}): RegistryWallet[] {
+  const isSSR = typeof window === "undefined";
+  const aptosConnectWallets = !isSSR && aptosConnect ? [
+    new AptosConnectGoogleWallet(aptosConnect),
+    new AptosConnectAppleWallet(aptosConnect),
+  ] : [];
+  return [
+    ...aptosConnectWallets,
+    petra,
```

Comment:
we are missing rimosafe and okx
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/registry.ts#L44-L55